src/Makefile

```diff
@@ -154,7 +145,6 @@ full:
 	make depend
 	make tags
 	make model
-	make disagg
```

Comment:
It was removed when @tbohn removed the `user_def.h` header file.  It is now specified in the global parameter file as `OUTPUT_FORCE = TRUE`.